lib/components/url_view/url_view.dart

```diff
@@ -40,6 +40,17 @@ class UrlView {
       filterTags.removeWhere((tag) => tag == "");
       return filterTags.toSet();
     }
+
+    // backward compatible read for 'filter' which is now 'include-filter'
+    if (type == TagFilterType.include) {
+      queryFilterKey = 'filter';
+      if (uri.queryParameters.containsKey(queryFilterKey)) {
+        List<String> filterTags = uri.queryParameters[queryFilterKey].split(' ');
+        filterTags.removeWhere((tag) => tag == "");
+        return filterTags.toSet();
+      }
+    }
+
```

Comment:
We don't have any projects running at the moment - so this is actually a good time to fully replace `filter` with `include-filter`, so let's just not add this part. Thanks for picking up the TODO!

```suggestion
```